.ci/jenkins/build-test/Jenkinsfile

```diff
@@ -21,6 +21,71 @@ pipeline {
                      required: true)
     }
     stages {
+        stage('Scan for possible secrets'){
+            steps{
+                sh label: "Secrets scan",
+                   script: '''
```

Comment:
It would be great to see this extracted into an external shell script file for ease of maintenance and integration with other pipelines, if needed.

Reply:
Done...